.examples/nginx/matomo.conf

```diff
@@ -0,0 +1,90 @@
+upstream php-handler {
+	server app:9000;
+}
+
+server {
+	listen 80;
+
+	add_header Referrer-Policy origin; # make sure outgoing links don't show the URL to the Matomo instance
+	root /var/www/html; # replace with path to your matomo instance
```

Comment:
From my tests, this is not true, php in matomo instance expect files to be at `/var/www/html`. Meaning that if you change the location of your matomo instance to `/var/www/html/matomo` for example. You will change the root in the server block to `/var/www/html/matomo` but php won't find the index.php file. You will have to add root `/var/www/html` inside the location block anywhere you use fastcgi_pass.

In code:

```
    root /var/www/html/matomo;

    index index.php;

    # make sure outgoing links don't show the URL to the Matomo instance
    add_header Referrer-Policy origin; 

    ## only allow accessing the following php files
    location ~ ^/(index|matomo|piwik|js/index).php {
        root /var/www/html;
        include             fastcgi_params;
        fastcgi_param HTTP_PROXY ""; # prohibit httpoxy: https://httpoxy.org/
        fastcgi_pass matomo:9000;
        fastcgi_param   SCRIPT_FILENAME    $document_root$fastcgi_script_name;
        fastcgi_param   SCRIPT_NAME        $fastcgi_script_name;
    }

```

Reply:
I just took the config from https://github.com/matomo-org/docker/issues/156#issuecomment-502367860.
This is just an example and this may not work with customized setups.